tests/t310-basis.h

```diff
@@ -0,0 +1,42 @@
+void buildmats(CeedScalar *qref, CeedScalar *qweight, CeedScalar *interp,
```

Comment:
Generally good practice to make this `static`. (At some point I may make it possible to build the tests into a library or to build one executable instead of needing to link dozens of separate executables. Not important yet though.)

Reply:
Done